docs/css/style.scss

```diff
@@ -1,22 +1,31 @@
 ---
 ---
 
+$font-stack: 'Karla', Helvetica, sans-serif;
+$primary-color: #10A882;
+
 html {
     box-sizing: border-box;
     font-size: 1rem;
-    font-family: "freight-sans-pro", "Freight Sans Pro", "Verdana", "Arial", sans-serif;
+    // font-family: "freight-sans-pro", "Freight Sans Pro", "Verdana", "Arial", sans-serif;
```

Comment:
[non-blocking] Do we need to keep this comment?

Reply:
No need! Thanks!